relative-paths.test.js

```diff
@@ -0,0 +1,42 @@
+/* global test, expect, afterEach */
```

Comment:
This files belongs into the src folder. Even tests are sources.

Reply:
So, in general the reason I have this `.js` file is more from my lack of familiarity with TypeScript. I am more than happy to have this file be in `src`, but relative to the problem I was trying to solve, I didn't wanna jump through hoops of TypeScript fixing while trying to get the gist of this thing working. I think specifically I was running into some issues trying to use `path` -- this evening I'll try and regroup and clarify my intentions in this PR.